tests/unit/bokeh/models/test_axes.py

```diff
@@ -40,6 +40,11 @@ def test_ticker_accepts_number_sequences() -> None:
     assert isinstance(a.ticker, FixedTicker)
     assert a.ticker.ticks == [-10, 0, 10, 20.7]
 
+def test_axis_label_accepts_math_text() -> None:
+    a = bma.Axis(axis_label=MathText(r"\sin(x)"))
+    assert isinstance(a.axis_label, MathText)
+    assert a.axis_label.text == r"\sin(x)"
```

Comment:
@mattpap @tcmetzger looking at `axis_label.text` here, any thoughts about a property name that makes it clear just from this usage that this is specifically the *math* text of a `MathText` label?

Reply:
How about `MathText.markup` or `MathText.math_markup` as a property name? This assumes that we want to support styles other than LaTeX (like MathML or AsciiMath) with the same property in the future - otherwise, we could also simply use  `MathText.latex`.

On the other hand, it might be nice for a `MathText` object to maintain compatibility with a non-LaTeX text object. For example, if I have some non-LaTeX `a.axis_label.text="foo"` and want to change my string to `r"$\sin(foo)$"`, I currently just need to create a `MathText` object (or Bokeh might even do that for me), while being able to leave all mentions of `.text` in place and not having to change them all to `.math_text` (or `._math_markup`, etc.).

Reply:
Let's just leave it as-is for now, and see how things seem after more work is done.